tests/test_data/plot_instant_air_temp_collapse.yaml

```
@@ -0,0 +1,36 @@
+name: Extract Instant Air Temperature
+description: |
+  Extracts out the instantaneous 1.5m air temperature from a file and writes it
+  to a new one.
```

```suggestion
name: Plot average air temperature
description: |
  Plots the mean 1.5m air temperature over an area.
```
Update description. This file probably also wants renaming.